geometry_msgs/Pose2D belief
geometry_msgs/Pose2D[] particles
geometry_msgs/Pose2D best_estimation
float64 best_estimation_weight
